docs/api/v2/node.md

```diff
@@ -46,19 +46,12 @@ method unsubscribe*(w: WakuNode, contentFilter: ContentFilter)
   ## Status: Not yet implemented.
   ## TODO Implement.
 
-method publish*(w: WakuNode, topic: Topic, message: Message)
-  ## Publish a `Message` to a PubSub topic.
+method publish*(w: WakuNode, topic: Topic, message: WakuMessage)
```

Comment:
Looks fine for now but eventually we should probably get something where topic is optional and some default (set at init?) is used. Or something better...

Reply:
Wouldn't this be a convenience function defined in e.g. status-waku? E.g. a partial function defined by config default topic.